### ISSUE TITLE
Dead humans can still be ripped for hearts

### ISSUE THREAD

**Issue by XSlayer300** [tgstation/TerraGov-Marine-Corps#8461]
### Description

Instead of husks, you can rip out hearts from humans even though they can be revived. While this isn't really noticeable, it brings to great effect in the Civil War gamemode.

### Test Merges

iirelevant, buuut.
#8436: 'Fix language holder harddel [DNM]' by BraveMole at commit eb7cbacae1
#8445: 'Revert "Makes buckshot a bit more effective mid range"' by TiviPlus at commit 233bd5c236
#8444: 'Asay and msay pings' by TiviPlus at commit ea72d38296

### Reproduction Steps

Basically, just rip out the heart of a marine with a knife, you can do so.

### Screenshots

![image](https://user-images.githubusercontent.com/35091844/135288257-ac25efed-e1b6-4bd7-bcfc-89e9eefa2bbc.png)

**Comment by BraveMole** [comment]
You can remove the heart if there is no ghost or undefib. So i guess the people just disconnect, which makes them heart rippeable
